cygnus-ngsi/src/main/java/com/telefonica/iot/cygnus/sinks/NGSIMySQLSink.java

```diff
@@ -384,11 +384,17 @@ public String getValuesForInsert() {
             
                 while (it.hasNext()) {
                     ArrayList<String> values = (ArrayList<String>) aggregation.get((String) it.next());
+                    String value = values.get(i);
+                    if (!value || value.equals("")) {
+                        value = "NULL";
+                    } else {
+                        value = "'" + value + "'";
+                    }
                     if (first) {
-                        valuesForInsert += "'" + values.get(i) + "'";
+                        valuesForInsert += value;
                         first = false;
                     } else {
-                        valuesForInsert += ",'" + values.get(i) + "'";
+                        valuesForInsert += "," + value;;
```

Comment:
extra semicolon?

Reply:
fixed in 007a052b59e3d7e08ebb3932fa1e26f82e1b2e51